src/lib/rest/StringFilter.cpp

```diff
@@ -846,6 +892,7 @@ bool StringFilter::parse(const char* q, std::string* errorStringP)
     }
 
     str = NULL;  // So that strtok_r continues eating the initial string
+    item.patternValueToBeFreed = false;
```

Comment:
For reference, the constructor code is here: https://github.com/telefonicaid/fiware-orion/pull/2081/files#diff-ef659d8f6e392bb5d95fdfbcd0997815R46